ts/src/zonda.ts

```diff
@@ -244,6 +244,8 @@ export default class zonda extends Exchange {
                 },
             },
             'options': {
+                'fetchTickerMethod': 'fetchTickerV2', // fetchTickerV1, fetchTickerV2
```

Comment:
I got a bit confused with the v1 and v2, expecting it to be the api version.
Would it make sense to call them pass the endpoint method here: `v1_01PublicGetTradingTickerSymbol` or `v1_01PublicGetTradingStatsSymbol`?
Or call them `ticket` and `stats`